packages/client/src/pages/Game.tsx

```diff
@@ -72,20 +71,11 @@ export const Game = ({ gameState, gameScore }: { gameState: GameState; gameScore
   const playCardOnTable = () => {
     if (activePlayerCard) {
       console.log('Emit: Card is going to the table');
-      gameIO.emit(GameEvent.UpdateState, roomName, {
-        ...gameState,
-        activePlayer: opponent.username,
-        players: [
-          opponent,
-          {
-            ...player,
-            hand: player.hand.filter((c) => cardKey(c) !== activePlayerCard),
-          },
-        ],
-        table: [...gameState.table, fromCardKey(activePlayerCard)],
+      gameIO.emit(GameEvent.PlayerAction, roomName, {
+        action: PlayerActionType.PlayOnTable,
+        playerName: player.username,
+        card: activePlayerCard,
       });
-      // TODO Might need to add a timeout here to keep the space of the card and avoid flickering
-      togglePlayerActiveCard(null);
```

Comment:
so how do you clear active cards now?

Reply:
Hm... That was the one setting the active card... Ok, moved it back!